src/app/workouts/page.tsx

```
@@ -0,0 +1,416 @@
+    const completedDate = new Date(date)
+    const diffMs = now.getTime() - completedDate.getTime()
+    const diffDays = Math.floor(diffMs / (1000 * 60 * 60 * 24))
+    
+    if (diffDays === 0) return 'Today'
+    if (diffDays === 1) return 'Yesterday'
+    if (diffDays < 7) return `${diffDays} days ago`
+    return completedDate.toLocaleDateString('en-US', { month: 'short', day: 'numeric' })
+  }
+  
+  // Helper function to count unique exercises in a workout
+  const countUniqueExercises = (workout: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)